ipfx/nwb_reader.py

```diff
@@ -71,9 +78,8 @@ def read_sweep_from_source(source):
                 real_sweep_number = timeseries.attrs["sweep_number"]
 
         if assumed_sweep_number is not None and assumed_sweep_number != real_sweep_number:
-            warnings.warn("Sweep number mismatch (real: {} vs assumed: {}"
-                          " in file {}".format(real_sweep_number,
-                                               assumed_sweep_number, self.nwb_file))
+            warnings.warn("Sweep number mismatch (real: {} vs assumed: {})".format
+                          (real_sweep_number, assumed_sweep_number))
```

Comment:
I would use the percent formatting for log messages.

See:
https://reinout.vanrees.org/weblog/2015/06/05/logging-formatting.html

Reply:
Does this equally apply to the warnings module?

Reply:
I'm not sure :/

Reply:
Thanks for pointing this issue out about the logging module. I probably will merge for now, since fixing the string formatting across many files would be best to address as a separate issue.